tests/tests/fv3_gfs_v16_dtend

```diff
@@ -0,0 +1,92 @@
+###############################################################################
+#
+#  FV3 CCPP GFS v16 compiled with 32-bit dynamics test in DEBUG mode
```

Comment:
As far as I can see, this test is not run in DEBUG mode, please fix line 3 and 7. Also add information that this test runs with extended/full tendencies.